webservices/tasks/legal_docs.py

```diff
@@ -68,15 +63,21 @@ def refresh_most_recent_legal_doc():
 
 def refresh_most_recent_aos(conn):
     """
-    When found any modified AO within 8 hours, relead all AOs starting the earliest AO.
+    When found modified AO(s) within 8 hours,
+    reload all AO(s) starting the earliest AO for each modified AO.
     """
-    logger.info(" Checking for recently modified AO...")
-    row = conn.execute(RECENTLY_MODIFIED_STARTING_AO).first()
-    if row:
-        logger.info(" Recently modified AO %s found at %s", row["ao_no"], row["pg_date"])
-        load_advisory_opinions(row["ao_no"])
-    else:
+    logger.info(" Checking for recently modified AOs...")
+    rs = conn.execute(RECENTLY_MODIFIED_AOS)
+    row_count = 0
+    for row in rs:
+        if row:
+            row_count += 1
+            logger.info(" Recently modified AO %s found at %s", row["ao_no"], row["pg_date"])
+            load_advisory_opinions(row["case_no"])
```

Comment:
On line#76, could you replace `load_advisory_opinions(row["case_no"]`) with  row ["ao_no"]?